__WTF/Scripts/Weapons/Health.cs

```diff
@@ -174,9 +178,10 @@ public virtual void Damage(HealthEventArgs e)
 			{
```

Comment:
Should we add a check...if dying, return early. Same thing in the Heal method.

Reply:
done 👍